katcp/testutils.py

```diff
@@ -792,7 +796,12 @@ def assert_request_succeeds(self, requestname, *params, **kwargs):
         informs_args_equal = kwargs.get("informs_args_equal")
 
         if args_echo:
-            args_equal = [str(p) for p in params]
```

Comment:
This could also be fixed at line 785 see https://github.com/ska-sa/katcp-python/pull/241/files#diff-98b8b54ad65a0b5eb81c30e4ff66fa9aL785